src/growthbook.ts

```diff
@@ -127,7 +119,6 @@ export const features = {
   }),
   dailyDigest: new Feature('daily_personalized_digest', {
     ...digestFeatureBaseConfig,
-    templateId: 'd-925d2759ddd641f99220b3c7c6836458',
```

Comment:
Digests now have the same template ID by default?

Reply:
yes. nimrod asked to merge the templates together